audit_report/3.1.0/changes/topics/cli.yml

```diff
@@ -68,4 +68,4 @@ patches:
 # Add PSK-support to TLS client/server CLI tools  (@reneme)
 - pr: 3552  # https://github.com/randombit/botan/pull/3552
   merge_commit: 9bb052c31540b336ab6b0008e64382fdcd438bb9
-  classification: unspecified
+  classification: info
```

Comment:
info because it just adds this functionality in CLI? Otherwise a feature like this is probably rather relevant, isn't it?